src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -2982,3 +2982,98 @@ def __init__(self, name, group, location):
                  checks=[
                      JMESPathCheck('length(@)', 0)
                  ])
+
+
+class SqlVirtualClusterMgmtScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer(random_name_length=17, name_prefix='clitest')
+    def test_sql_virtual_cluster_mgmt(self, resource_group, resource_group_location):
+        vnet_name = 'vcCliTestVnet'
+        subnet_name = 'vcCliTestSubnet'
+        route_table_name = 'vcCliTestRouteTable'
+        route_name = 'vcCliTestRoute'
+
+        # Create and prepare VNet and subnet for new virtual cluster
+        self.cmd('network route-table create -g {} -n {}'.format(resource_group, route_table_name))
```

Comment:
Won't ask that you refactor this, but for future reference, there's a more concise way of writing these tests:
```Python
self.kwargs.update({
  'vnet': 'vcCliTestVnet',
  'routetable': 'vcCliTestRouteTable',
  … # etc
})
self.cmd('network route-table create -g {rg} -n {routetable}', checks=[
  self.check('name', '{routetable}'),
  self.check('resourceGroup': '{rg}'),
  … # etc.
])
```

See the following link for more examples:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_tests.md#sample-5-get-more-from-resourcegrouppreparer

Reply:
Thanks for the suggestion, I modified the test.